tools/mydumper.md

```diff
@@ -0,0 +1,46 @@
+---
+title: mydumper Instructions 
+summary: Use mydumper to export data from TiDB.
+category: tools
+---
+
+# mydumper Instructions
+
+## What is mydumper?
+
+`mydumper` is a fork of the [mydumper](https://github.com/maxbube/mydumper) project with additional functionality specific to TiDB. It is the recommended method to use for logical backups of TiDB.
+
+[Download the Binary](http://download.pingcap.org/tidb-enterprise-tools-latest-linux-amd64.tar.gz).
+
+## What enhancements does this contain over regular mydumper?
+
```

Comment:
For the following sentences, we had better use "Uses", "Includes", and "Allows" instead of "Use", "Include" and "Allow", because these sentences are descriptive (describe something) rather than imperative (tell users to do something).
Reference: "These comments should be descriptive ("Opens the file") rather than imperative ("Open the file"); the comment describes the function, it does not tell the function what to do." on https://google.github.io/styleguide/cppguide.html